src/resolvers/utils.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import http from 'http';
+import https from 'https';
+
+export async function fetchHttpImage(url: string, options = {}): Promise<Buffer> {
+  return (
+    await axios({
+      url,
+      ...{
+        responseType: 'arraybuffer',
+        timeout: 5e3,
+        ...options
+      }
+    })
+  ).data;
+}
```

Comment:
I mean like this
```typescript
export async function fetchHttpImage(url: string, options = {}): Promise<Buffer> {
  return (
    await axios({
      url,
      ...{
        responseType: 'arraybuffer',
        timeout: 5e3,
        ...options,
        ...axiosDefaultParams
      }
    })
  ).data;
}
```

we can also make it optional based on some param passed to `fetchHttpImage`

Reply:
Only problem is future generations may not remember about `axiosDefaultParams` 😋

Reply:
`fetchImage` does not need keepalive, as I don’t think it has benefits when the url is not from same host. We could still uses it there though